app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file = command.split()[1]
+    new_file = command.split()[2]
+    if file != new_file:
```

Comment:
It's also generally better not to put function's main logic into an `if` block, even though in this case it doesn't make much difference. It would be better to do it like this:
```python
if file == new_file:
    return
...
```

You may also want to check if `command` == `cp`.

Reply:
fixed